configs/marketplace/base-goerli.json

```
@@ -1,35 +1,782 @@
-	{
-		"author": "Uniswap",
-		"id": "uniswap",
-		"title": "Uniswap",
-		"logo": "https://www.gitbook.com/cdn-cgi/image/width=288,dpr=2.200000047683716,format=auto/https%3A%2F%2Ffiles.gitbook.com%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252F-Lq1XoWGmy8zggj_u2fM%252Fuploads%252FT8RR0UqqvH6wAVfaaM4f%252FUniswap.png%3Falt%3Dmedia%26token%3D797851a0-2d80-4f7b-8d80-f9ba664c0b3d",
-		"categories": ["tools"],
-		"shortDescription": "Uniswap is a cryptocurrency exchange which uses a decentralized network protocol.",
-		"site": "https://docs.uniswap.org/",
-		"description": "Uniswap is a cryptocurrency exchange which uses a decentralized network protocol.",
-		"url": "https://app.uniswap.org/swap"
-	},
-	{
-		"author": "Aave",
-		"id": "aave",
-		"title": "Aave",
-		"logo": "https://www.gitbook.com/cdn-cgi/image/width=288,dpr=2.200000047683716,format=auto/https%3A%2F%2Ffiles.gitbook.com%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252F-Lq1XoWGmy8zggj_u2fM%252Fuploads%252FaNQY8jHNue30P2caV79v%252FAave.png%3Falt%3Dmedia%26token%3Dda9ea09b-a5d8-4bf1-bdfe-41837eebdd03",
-		"categories": ["tools"],
-		"shortDescription": "Aave is a decentralised non-custodial liquidity market protocol where users can participate as suppliers or borrowers. Suppliers provide liquidity to the market to earn a passive income, while borrowers are able to borrow in an overcollateralised (perpetually) or undercollateralised (one-block liquidity) fashion.",
-		"site": "https://docs.aave.com/faq/",
-		"description": "Aave is a decentralised non-custodial liquidity market protocol where users can participate as suppliers or borrowers. Suppliers provide liquidity to the market to earn a passive income, while borrowers are able to borrow in an overcollateralised (perpetually) or undercollateralised (one-block liquidity) fashion.",
-		"url": "https://staging.aave.com/"
-	},
-	{
-		"author": "Blockscout",
-		"id": "token-approval-tracker",
-		"title": "Token Approval Tracker",
-		"logo": "https://approval-tracker.apps.blockscout.com/icon-192.png",
-		"categories": ["security", "tools"],
-		"shortDescription": "Token Approval Tracker shows all approvals for any ERC20-compliant tokens and NFTs and lets to revoke them or adjust the approved amount.",
-		"site": "https://docs.blockscout.com/for-users/blockscout-apps/token-approval-tracker",
-		"description": "Token Approval Tracker shows all approvals for any ERC20-compliant tokens and NFTs and lets to revoke them or adjust the approved amount.",
-		"url": "https://approval-tracker.apps.blockscout.com/"
-	}
+  {
+    "author": "0x",
+    "id": "0x",
+    "title": "0x",
+    "logo": "https://www.gitbook.com/cdn-cgi/image/width=32,dpr=2.200000047683716,format=auto/https%3A%2F%2Ffiles.gitbook.com%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252F-Lq1XoWGmy8zggj_u2fM%252Fuploads%252FxVr4QpRAlyhTrqmt9KPb%252FPlaceholder.png%3Falt%3Dmedia%26token%3D3c1c5532-a0e5-4e79-b36e-4466a0fffbf6",
+    "categories": "infra & dev tooling",
```

here and for other apps categories should be an array
```suggestion
    "categories": ["infra & dev tooling"],
```